structuralalignment/superposition/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
```

Comment:
Do we need that file?

Reply:
Unfortunately yes. It marks a directory as a (sub)package and allows us to document the whole subpackage through the module docstring, if needed.